File: benchmark/Fib.java
public final class Fib {
    private Fib() {}

    private static int fib(int value) {
        if (value <= 1) {
            return value;
        }
        return fib(value - 1) + fib(value - 2);
    }

    public static void main(String[] args) {
        long started = System.nanoTime();
        int result = fib(35);
        long elapsedMilliseconds = (System.nanoTime() - started) / 1_000_000L;

        System.out.println("RESULT=" + result);
        System.out.println("TIME_MS=" + elapsedMilliseconds);
    }
}

File: benchmark/fib.rb
def fib(n)
  return n if n <= 1
  fib(n - 1) + fib(n - 2)
end

start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
result = fib(35)
end_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)

elapsed_ms = ((end_time - start_time) * 1000).to_i
puts "RESULT=#{result}"
puts "TIME_MS=#{elapsed_ms}"
